xplique/attributions/sobol/sobol_attribution_method.py

```diff
@@ -0,0 +1,194 @@
+"""
+Sobol Attribution Method explainer
+"""
+
+from enum import Enum
+
+import cv2
+import numpy as np
+import tensorflow as tf
+
+from ...types import Callable, Union, Optional, Tuple
+from ...commons import batch_tensor, repeat_labels
+from ..base import BlackBoxExplainer, sanitize_input_output
+from .estimators import SobolEstimator, JansenEstimator
+from .sampling import Sampler, TFSobolSequence
+from .perturbations import amplitude, inpainting, blurring
+
+
+class PerturbationFunction(Enum):
+    """
+    Sobol Perturbation function interface.
+    """
+    INPAINTING = inpainting
+    BLURRING   = blurring
+    AMPLITUDE  = amplitude
+
+    @staticmethod
+    def from_string(perturbation_function: str) -> 'PerturbationFunction':
+        """
+        Restore a perturbation function from a string.
+
+        Parameters
+        ----------
+        perturbation_function
+            String indicating the perturbation function to restore, e.g 'inpainting', 'blurring'.
```

Comment:
could be changed to:
String indicating the perturbation function to restore: must be one of 'inpainting', 'blurring' or 'amplitude'.

Reply:
true, fixed ;)